provider/aws/processes_test.go

```diff
@@ -264,8 +264,12 @@ func TestProcessRunDetached(t *testing.T) {
 
 func TestProcessStop(t *testing.T) {
 	provider := StubAwsProvider(
-		cycleProcessListTasksRunning,
```

Comment:
why did you remove these?

Reply:
Because I changed the method to get the process, before it only gets running and stopped, it didn't get the process from the BuildInstance.
The `cycleProcessListTasksRunning` and `cycleProcessListTasksStopped` don't satisfy the appTaskARNs request, I had to change the stub processes.